Formula/brev.rb

```diff
@@ -0,0 +1,18 @@
+class Brev < Formula
+  desc "CLI tool for managing workspaces provided by brev.dev"
+  homepage "https://docs.brev.dev"
+  url "https://github.com/brevdev/brev-cli/archive/refs/tags/v0.6.17.tar.gz"
+  sha256 "cf69be0558cd87ca5838372aeca6b85fe937f1bd8eb44c03a88355ca088aa9ea"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-X github.com/brevdev/brev-cli/pkg/cmd/version.Version=v#{version}"
+    system "go", "build", *std_go_args(ldflags: ldflags)
+  end
+
+  test do
+    system "#{bin}/brev", "healthcheck"
```

Comment:
Great, but we don't need interpolation here.
```suggestion
    system bin/"brev", "healthcheck"
```